library/xml/src/modules/XML.rb

```diff
@@ -300,8 +300,7 @@ def add_element(doc, metadata, parent, contents)
         raise XMLInvalidKey, key unless key.is_a?(::String)
 
         value = contents[key]
-        type_attr = metadata["typeNamespace"] ? "config:" : ""
```

Comment:
maybe we should document that typeNamespace is now ignored and is deprecated as it does not affect anything?

Reply:
Will do